AudioStream.h

```diff
@@ -41,8 +41,8 @@
 #define __IMXRT1062__  // pretend this is a Teensy 4.x
 #define FLASHMEM
 #define ARM_DWT_CYCCNT (m33_hw->dwt_cyccnt)
-#define __disable_irq(...) noInterrupts()
-#define __enable_irq(...)    interrupts()
+//#define __disable_irq(...) noInterrupts()
```

Comment:
Why removing the macros? With these the code would require less changes to work for the RP2350.